src/models/propulsion/FGTurbine.h

```diff
@@ -246,7 +249,7 @@ class FGTurbine : public FGEngine
   double MilThrust;        ///< Maximum Unaugmented Thrust, static @ S.L. (lbf)
   double MaxThrust;        ///< Maximum Augmented Thrust, static @ S.L. (lbf)
   double BypassRatio;      ///< Bypass Ratio
-  std::unique_ptr<FGParameter> TSFC;   ///< Thrust Specific Fuel Consumption (lbm/hr/lbf)
+  std::unique_ptr<FGTSFC> TSFC;        ///< Thrust Specific Fuel Consumption (lbm/hr/lbf)
```

Comment:
Keep the type of member `TSFC` as `FGParameter` because it will contain either an `FGFunction` or a `FGTSFC` instance.
```suggestion
  std::unique_ptr<FGParameter> TSFC;   ///< Thrust Specific Fuel Consumption (lbm/hr/lbf)
```